service_capacity_modeling/models/org/netflix/ddb.py

```diff
@@ -0,0 +1,584 @@
+            "data_transfer_gib": data_transfer_plan.total_data_transfer_gib,
+        }
+        requirement_context[
+            "replicated_write_capacity_units"
+        ] = write_plan.replicated_write_capacity_units
+        requirement = CapacityRequirement(
+            requirement_type="dynamo-regional",
+            core_reference_ghz=0,
+            cpu_cores=certain_int(0),
+            disk_gib=certain_float(storage_plan.total_data_storage_gib),
+            context=requirement_context,
+        )
+        dynamo_services = [
+            ServiceCapacity(
+                service_type="dynamo.standard",
+                annual_cost=(
+                    write_plan.total_annual_write_cost
+                    + read_plan.total_annual_read_cost
+                    + storage_plan.total_annual_data_storage_cost
+                    + data_transfer_plan.total_annual_data_transfer_cost
+                ),
+                service_params={
+                    "read_capacity_units": read_plan.read_capacity_units,
+                    "write_capacity_units": (
+                        write_plan.write_capacity_units
+                        + write_plan.replicated_write_capacity_units
+                    ),
+                },
+            ),
+            ServiceCapacity(
+                service_type="dynamo.backup",
+                annual_cost=backup_plan.total_annual_backup_cost,
+            ),
+        ]
+        dynamo_costs = {
+            "dynamo.regional-writes": write_plan.total_annual_write_cost,
+            "dynamo.regional-reads": read_plan.total_annual_read_cost,
+            "dynamo.regional-storage": storage_plan.total_annual_data_storage_cost,
+        }
+
+        dynamo_costs[
+            "dynamo.regional-transfer"
+        ] = data_transfer_plan.total_annual_data_transfer_cost
+
+        dynamo_costs["dynamo.data-backup"] = backup_plan.total_annual_backup_cost
+
+        clusters = Clusters(
+            annual_costs=dynamo_costs,
+            zonal=[],
+            regional=[],
+            services=dynamo_services,
+        )
+
+        return CapacityPlan(
+            requirements=Requirements(regional=[requirement]),
+            candidate_clusters=clusters,
+        )
+
+    @staticmethod
+    def description():
+        return "Netflix Streaming DynamoDB Model"
+
+    @staticmethod
+    def extra_model_arguments_schema() -> Dict[str, Any]:
+        return NflxDynamoDBArguments.schema()
+
+    @staticmethod
+    def run_hardware_simulation() -> bool:
+        return False
+
+    @staticmethod
+    def default_desires(user_desires, extra_model_arguments: Dict[str, Any]):
+        acceptable_consistency = {
+            "same_region": {
+                None,
+                AccessConsistency.best_effort,
+                AccessConsistency.eventual,
+                AccessConsistency.read_your_writes,
+                AccessConsistency.never,
```

Comment:
I believe DDB can do up to `serializable` (essentially all consistency levels) in the same region

Reply:
let me update this